src/main/java/frc/robot/SetPointConstants.java

```diff
@@ -0,0 +1,16 @@
+package frc.robot;
+
+public class SetPointConstants {
+    public static final double coralIntakeHeight = 0.0;
```

Comment:
As a general principle, constants should always have uppercase naming. In this case, `CORAL_INTAKE_HEIGHT`.